modules/ossm-config-otel-kiali.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/kiali/ossm-kiali-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-otel-kiali_{context}"]
+= Configuring {DTProductName} with Kiali
+
+When {KialiProduct} is properly configured with {DTProductName}, you can view distributed traces in the Kiali console. These traces provide insight into the communication between services within the service mesh, helping you understand how requests are flowing through your system and where potential issues might be.
+
+.Prerequisites
+
+* {SMProductName} is installed.
+
+* You configured {DTShortName} with {SMProductName}.
+
+.Procedure
+
+. Setup the Kiali custom resource to access traces from the {TempoName} frontend:
++
+.Example Tempo frontend
+[source,yaml]
+----
+external_services:
+  grafana:
+    enabled: true #<1>
+    in_cluster_url: "http://grafana-istio-system.apps-crc.testing/"
+    url: "http://grafana-istio-system.apps-crc.testing/"
+  tracing:
+    enabled: true
+    provider: tempo
+    use_grpc: false
+    in_cluster_url: http://tempo-sample-query-frontend.tempo:3200
+    url: 'https://tempo-sample-query-frontend-tempo.apps-crc.testing'
+    tempo_config: #<2>
+      org_id: "1"
+      datasource_uid: "a8d2ef1c-d31c-4de5-a90b-e7bc5252cd00"
+----
+<1> Required so that you can see the "View in Tracing" link from the **Traces** tab.
+<2> Required so that you can see the "View in Tracing" link from the **Traces** tab and redirect to the proper Tempo `datasource`.
```

Comment:
```suggestion
<1> Required so that you can see the "View in Tracing" link from the *Traces* tab.
<2> Required so that you can see the "View in Tracing" link from the *Traces* tab and redirect to the proper Tempo `datasource`.
```

See additional comment re formatting items in the UI